Yamb/YambDataSource.swift

```diff
@@ -198,7 +220,8 @@ class YambDataSource {
     }
     
     func clear(indexPath: IndexPath) {
-        if let field = fieldsDict[indexPath.section]?[indexPath.item] {
+        if let field = fieldsDict[indexPath.section]?[indexPath.row] {
```

Comment:
why did you switch to .row?

Reply:
I wasn't sure what the difference was between .item and .row and was testing out, I changed it back to .item